.dockerignore

```diff
@@ -1,5 +1,82 @@
+**/bin/**
```

Comment:
*NO PR*

we just need to ignore more since we're really building from source during the docker build